docs/Caching.md

```diff
@@ -0,0 +1,17 @@
+# Backend Caching 
```

Comment:
To make this show up on docs.cbioportal.org add it to https://github.com/cBioPortal/cbioportal/blob/master/docs/SUMMARY.md and https://github.com/cBioPortal/cbioportal/blob/master/docs/README.md

Reply:
done